src/modules/map/components/cesium/CesiumInternalLayer.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div>
+        <CesiumWMTSLayer
+            v-if="layerConfig.type === LayerTypes.WMTS && !layerConfig.isExternal"
+            :wmts-layer-config="layerConfig"
+            :preview-year="previewYear"
+            :projection="WEBMERCATOR"
```

Comment:
don't know if it would make sense to directly use the same logic as the one for OL, meaning :
```
:projection="layerConfig.isExternal ? WEBMERCATOR : LV95"
```
Some of our WMTS layers do not behave totally correctly when requested as `EPSG:3857` or `EPSG:4326`

Reply:
maybe we can investigate and try to use `WebMapServiceImageryProvider` instead of `UrlTemplateImageryProvider` if LV95 required for WMTS

Reply:
let's keep mercator there then, it's something we should investigate if we want to have LV95 projection for 3D in the future (I doubt it's a good idea anyway, for 3D at least)